notesapi/v1/tests/test_models.py

```diff
@@ -47,3 +48,12 @@ def test_must_have_fields_create(self):
             with self.assertRaises(ValidationError):
                 note = Note.create(payload)
                 note.full_clean()
+
+    def test_default_tags_value(self):
+        """ Test that a note without explicit tags has an empty list for the tags value """
+        payload = self.note_dict.copy()
+        payload.pop("tags")
+        note = Note.create(payload)
+        note.full_clean()
+        note.save()
+        self.assertEqual([], note.as_dict()["tags"])
```

Comment:
Must: I think you should also validate that `note.tags` is what you expect it to be; not just `note.as_dict()["tags"]`.